### PR TITLE
fix: clarification on 'interactive' argument in client docstring

### PR DESCRIPTION
**Pull request by efredriksson-modelon**

### CODE REVIEW

File: modelon/impact/client/client.py [hunk 1]
```diff
@@ -25,9 +25,19 @@ class Client:
...
 
         interactive --
-            True, if running an interactive python shell. Defaults to the value
-            specified by env variable 'MODELON_IMPACT_CLIENT_INTERACTIVE' if set else
-            False.
+            If True the client will prompt for an API key if no other login information
+            can be found. An API key entered for this prompt will be saved and re-used
+            next time the Client is instantiated. If False no prompt will be given if
+            no other login information can be found and login will be done as if no
+            API key was given (anonymous login).
```
Comment (maraberg) [inline modelon/impact/client/client.py:32]: Very good! 

One minor thing could be added: Where is key saved?

**Review by maraberg — COMMENTED** [review]
I think it is ok, 

I think that mentioning where key is stored could be valuable. There might be reasons for not doing this I am not aware of though.

**Review by maraberg — APPROVED** [review]
Looks good!

File: modelon/impact/client/client.py [hunk 2]
```diff
@@ -25,9 +25,19 @@ class Client:
             'http://localhost:8080/'.
 
         interactive --
-            True, if running an interactive python shell. Defaults to the value
-            specified by env variable 'MODELON_IMPACT_CLIENT_INTERACTIVE' if set else
-            False.
+            If True the client will prompt for an API key if no other login information
+            can be found. An API key entered for this prompt will be saved to disc
```
Comment (jacobtaxen) [inline modelon/impact/client/client.py:29]: disc -> disk

**Review by jacobtaxen — APPROVED** [review]
Looks good to me

**Pull request merged**